src/compositions/DisasterPosts/DisasterPosts.css

```diff
@@ -50,22 +50,58 @@
 
 .DisasterPosts .modal-details {
     display: block;
-    position: relative;
-    margin: auto;
+    position: fixed;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
     z-index: 6;
     width: 60%;
-    height: fit-content;
+    overflow-y: auto;
```

Comment:
If you want the overflow scrollbar to show, you just need a defined height. Overflow looks at the height to know when to add the scrollbar. 

You can just use a max-height declaration and that counts as a defined height. If the content is short, it'll collapse to fit the content. If the content is long enough to cause overflow, the max-height declaration will kick in and the overflow scrollbar will be applied.

You can probably do max-height 80 or 90% and it should be cool. 

Here's a quick jsbin to demonstrate: [https://jsbin.com/yuviropene/edit?html,css,output](https://jsbin.com/yuviropene/edit?html,css,output)

You can change the height of the p element to smaller and it'll collapse.